modules/azure/src/main/java/org/testcontainers/azure/AzureEventHubsEmulatorContainer.java

```diff
@@ -0,0 +1,138 @@
+package org.testcontainers.azure;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.LicenseAcceptance;
+
+/**
+ * Testcontainers implementation for Azure Eventhubs Emulator.
+ * <p>
+ * Supported image: {@code "mcr.microsoft.com/azure-messaging/eventhubs-emulator"}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>AMQP: 5672</li>
+ *     <li>Kafka: 9092</li>
+ * </ul>
+ */
+public class AzureEventHubsEmulatorContainer extends GenericContainer<AzureEventHubsEmulatorContainer> {
```

Comment:
We are not using the word Emulator on Azure module so let's be consistent.

```suggestion
public class AzureEventHubsContainer extends GenericContainer<AzureEventHubsEmulatorContainer> {
```

Reply:
Renamed. I assume you meant Azurite only because you are using the Emulator word in case of CosmosDB.